packages/flutter/lib/src/material/circle_avatar.dart

```diff
@@ -2,15 +2,17 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// ignore_for_file: public_member_api_docs
+
 /// @docImport 'chip.dart';
 /// @docImport 'color_scheme.dart';
 /// @docImport 'list_tile.dart';
 library;
 
-import 'package:flutter/widgets.dart';
+import 'dart:core';
+import 'dart:math';
 
-import 'constants.dart';
-import 'theme.dart';
+import 'package:flutter/material.dart';
```

Comment:
A few thoughts here:
- I believe the `dart:core` import is done automatically and doesn't need to be explicitly added.
- The [style guide](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#import-conventions) recommends always doing `import 'dart:math' as math;`
- Rather than importing the material library from within the material library, it's better to do relative imports.